src/soilbiogeochem/SoilBiogeochemLittVertTranspMod.F90

```diff
@@ -182,9 +182,6 @@ subroutine SoilBiogeochemLittVertTransp(bounds, num_soilc, filter_soilc,      &
       if ( use_c14 ) then
          ntype = ntype+1
       endif
```

Comment:
I remember this part of the code was confusing to me.  I feel like @slevisconsulting was using this with MIMICS to control some aspect related to the pools assumed to be in BGC vs. FATES_BGC cases regarding how the different models handle CWD?  Maybe this is not accurate, but I wanted to raise the point here.

Reply:
The ntype here refers to the different chemical elements, C, C13,C14 and N. Since  fates used to only do c-only, and now uses the other chemical species, we don't differentiate ntype anymore.